apps/console/app/lib/auth/stack-auth.ts

```
@@ -52,6 +53,14 @@ const authService = {
     });
     return apiKey.value;
   },
+
+  getAccessToken() {
+    const rawCookie = getCookie("stack-access");
+    if (!rawCookie) return;
```

Hiding an exception condition, should throw instead.

---

Done.